simplebgc/src/commands/read_params.rs

```diff
@@ -47,12 +47,15 @@ pub struct AxisRcParams {
 
 payload_rpy!(AxisRcParams, 8);
 
-#[derive(BitFlags, Copy, Clone, Debug, PartialEq)]
+#[derive(FromPrimitive, ToPrimitive, Copy, Clone, Debug, PartialEq)]
 #[repr(u8)]
 pub enum AxisRcMode {
-    Angle = 1 << 0,
-    Speed = 1 << 1,
-    Inverted = 1 << 2,
+    AngleRegular = 0,
+    Speed = 1,
+    AngleTracking = 2,
+    InvertedAngleRegular = 8,
+    InvertedSpeed = 9,
+    InvertedAngleTracking = 10,
```

Comment:
Consider specifying these numbers as binary literals (`0b0000`, `0b1000`) so it's clearer how they are related.

Reply:
done